proglogica/logica_teoria/aplicaciones.html

```diff
@@ -332,6 +334,66 @@ <h2 class="title"><strong>Consultas lógicas basadas en reglas</strong></h2>
+                        
+                        La manera de tratar la incertidumbre en la programación lógica es mediante la adición a esta 
+                        del concepto de probabilidad, generando en este caso la programación lógica probabilística. 
+                        Con la programación lógica probabilística se seguirá el mismo funcionamiento que en la 
+                        programación lógica, pero teniendo en cuenta que los hechos presentados no tienen por qué 
+                        ocurrir siempre, como ocurre en la vida real.
+                        
+                        </p>
+                        <ul>
+                            <li style="margin-left:2em">
+                                <p><font color="red">Problog</font> es un lenguaje de programación probabilístico que se basa en Prolog.
+                                    Desde una perspectiva de modelado, se puede considerar que un programa de Problog está compuesto por dos partes, 
+                                    una parte probabilística que define una distribución de probabilidad sobre los valores de verdad  y otra logica que 
+                                    deriva los valores de verdad utilizando mecanismos de razonamiento como en Prolog.
+                                    </p>
+                                <p>Ejemplo en Problog
+                                    El siguiente programa modela un pequeño grupo social, donde el tabaquismo de las personas es influenciado por sus amigos, 
+                                    e indirectamente por los amigos de los amigos.
+                                </p>    
+                                    0.4::asthma(X) :- smokes(X). <br>
+                                    0.3::smokes(X). <br>
+                                    0.2::smokes(X) :- friend(X,Y), smokes(Y). <br>
+                                    friend(1,2). friend(2,1). friend(2,4). friend(3,2). friend(4,2).<br>
+                                    <br>
+                                <p>
+                                    La principal tarea de inferencia abordada en ProbLog es la de calcular la probabilidad de que una consulta 
+                                    tenga éxito, por ejemplo ?- asthma(2) tiene éxito con una probabilidad de 0,15. Si añadimos asthma(3) como evidencia, 
+                                    la probabilidad condicional de ?- asthma(2) es de 0,19.
+                                </p>
+                                <p>
+                                    Problog2 es el sucesor de Problog, y es uno de los lenguajes lógicos probabilísticos 
+                                    más modernos. Este lenguaje además presenta grandes aplicaciones en el ámbito de la 
+                                    inteligencia artificial además que permite modelar y obtener respuestas rápidamente 
+                                    además de la capacidad de tratar conjuntos de datos muy grandes.  
```

Comment:
```suggestion
                                    pues cuenta con la capacidad de tratar conjuntos de datos muy grandes.  
```
Eliminación de repeticiones de la palabra "además".